packages/components/package.json

```diff
@@ -3,15 +3,16 @@
   "version": "1.3.0",
   "license": "MIT",
   "main": "dist/index.js",
+  "module": "dist/index.js",
   "types": "dist/index.d.ts",
-  "module": "dist/index.es.js",
   "scripts": {
-    "watch": "vite build --watch",
-    "build": "cross-env NODE_OPTIONS='--max-old-space-size=8192' vite build",
-    "build-decls": "tsc",
+    "build": "tsc",
+    "watch": "tsc -w",
+    "start": "tsc -w",
```

Comment:
Let's not add a `start` script here, since it's redundant with `watch`, we don't have a Nx declaration for it, and we also don't have a `start` script in `core` as of #1173:
```suggestion
```